tests/StreamWrapper/IncludeInterceptorTest.php

```diff
@@ -34,6 +34,7 @@
 
     public static function setUpBeforeClass()
     {
+        var_dump('Settign up');
```

Comment:
Should this be here?

Reply:
Woops, forgot to remove that during some debugging.